addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -2329,7 +2329,7 @@ export function getRangePosition(el, document, options = {}) {
         clonedRange.detach();
     }
 
-    if (!offset || offset.heigh === 0) {
+    if (!offset || offset.height === 0) {
```

Comment:
is it related to the fix?

Reply:
@shsa-odoo No, but I noticed that it may cause a bug in future and it is fixed in 16.0 and above.